.github/ISSUE_TEMPLATE/add_a_banner.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Add a banner
+about: Use this template for requesting top informational banner in all search
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+### Summary
+
+### Banner Text
+- Text:
+
+### Impact
+_Please include hard deadlines and date range for banner implementation_
+- Go live date:
+- Takedown date:
+
+### Priority recommendation
+
+- [ ] asap
+- [ ] within the next 3 weeks
+
+### Sudden Priority Justification
+_Required if "asap" is checked._
+
+### Implementation notes
+[Documentation on how to set new banner messages](https://github.com/pulibrary/all_search#announcement-messages)
```

Comment:
The documentation will be added in the Readme after we resolve the ticket to add the rake task and the feature flipper.